fenix/app/src/test/java/org/mozilla/fenix/tabstray/TabsTrayStateTest.kt

```diff
@@ -0,0 +1,208 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.tabstray
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import mozilla.components.support.test.libstate.ext.waitUntilIdle
+import mozilla.components.support.test.robolectric.testContext
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.MenuItem
+import org.mozilla.fenix.tabstray.ext.getMenuItems
+import org.mozilla.fenix.tabstray.ext.isSelect
+
+@RunWith(AndroidJUnit4::class)
+class TabsTrayStateTest {
+
+    @Mock
```

Comment:
Does this annotation still make sense if we're assigning the variable to `TabsTrayStore` in the setup?